daml/Marketplace/Custody.daml

```diff
@@ -33,11 +33,11 @@ template Service
       nonconsuming RequestOpenAccount : ContractId OpenAccountRequest
         with
           accountId : Id
-          observers : Set Party
-          ctrls : Set Party
+          observers : [Party]
+          ctrls : [Party]
```

Comment:
In another file you actually did the opposite :laughing: i.e. changed List -> set, which I do agree with. :smile:

Reply:
Ah, this is because Sets are a pain to work with from the UI. I thought passing a list to the choice, which is then transformed into a Set right away would be ok. What do you think? Not feeling strong about this one though. Maybe the new DA.Set is represented better in TS.